FhirToDataLake/src/Microsoft.Health.Fhir.Synapse.Core/Jobs/AzureStorageJobFactory.cs

```diff
@@ -0,0 +1,96 @@
+﻿// -------------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License (MIT). See LICENSE in the repo root for license information.
+// -------------------------------------------------------------------------------------------------
+
+using Microsoft.Extensions.Logging;
+using Microsoft.Health.Fhir.Synapse.Core.DataProcessor;
+using Microsoft.Health.Fhir.Synapse.Core.Jobs.Models;
+using Microsoft.Health.Fhir.Synapse.DataClient;
+using Microsoft.Health.Fhir.Synapse.DataWriter;
+using Microsoft.Health.JobManagement;
+using Newtonsoft.Json;
+
+namespace Microsoft.Health.Fhir.Synapse.Core.Jobs
+{
+    public class AzureStorageJobFactory : IJobFactory
+    {
+        private IQueueClient _queueClient;
```

Comment:
Will the `_queueClient` be re-assigned some where? Or maybe adding the `readonly` term makes more sense?